facebookads/objects.py

```diff
@@ -1019,21 +1019,21 @@ def get_transactions(self, fields=None, params=None):
 
     def get_conversion_stats(self, fields=None, params=None):
         """
-        Returns iterator over ConversionStats's associated with this account.
+        Returns ConversionStats object for this account.
         """
         return self.edge_object(ConversionStats, fields, params)
 
     def get_ad_campaign_conversion_stats(self, fields=None, params=None):
-        """Returns an AdCampaignConversionStats object for this account."""
-        return self.edge_object(
+        """Returns iterator over AdCampaignConversionStats objects associated this account."""
```

Comment:
This will cause our linter to fail the push, can you reduce this line to 80 characters?